documentation/modules/security/con-securing-kafka-authentication.adoc

```diff
@@ -95,6 +95,10 @@ mTLS listener configuration in the `Kafka` resource requires the following:
 * `tls: true` to specify TLS encryption and server authentication
 * `authentication.type: TLS` to specify the client authentication
 
+When a Kafka cluster is created by the Cluster Operator, it creates a new secret with the name `<cluster_name>-cluster-ca-cert`.
```

Comment:
`<cluster_name>-cluster-ca-cert` actually has only the public key. The private key is in `<cluster_name>-cluster-ca`.